tests/test_filtsmooth/test_gaussfiltsmooth/test_kalman.py

```diff
@@ -50,3 +50,127 @@ def test_info_dicts(setup):
 
     assert isinstance(info_dicts, list)
     assert len(posterior) == len(info_dicts)
+
+
+def test_kalman_smoother_high_order_ibm():
+    """The highest feasible order (without damping) is 10.
+
+    If this test breaks, someone played with the stable square-root
+    implementations: for instance,
+    """
+    regression_problem, statespace_components = filtsmooth_zoo.car_tracking(
+        model_ordint=10,
+        timespan=(0.0, 1e-3),
+        step=1e-6,
+        forward_implementation="sqrt",
+        backward_implementation="sqrt",
+    )
+    truth = regression_problem.solution
+
+    kalman = filtsmooth.Kalman(
+        statespace_components["dynamics_model"],
+        statespace_components["measurement_model"],
+        statespace_components["initrv"],
+    )
+
+    posterior, _ = kalman.filtsmooth(regression_problem)
+
+    filtms = posterior.filtering_posterior.states.mean
+    smooms = posterior.states.mean
+
+    filtms_rmse = np.mean(np.abs(filtms[:, :2] - truth[:, :2]))
+    smooms_rmse = np.mean(np.abs(smooms[:, :2] - truth[:, :2]))
+    obs_rmse = np.mean(np.abs(regression_problem.observations - truth[:, :2]))
+
+    assert smooms_rmse < filtms_rmse < obs_rmse
+
+
+def test_kalman_smoother_high_order_ibm_with_process_noise_damping_crazy_small_steps():
+    """If we add a small damping factor to the computation of the Cholesky factor of the
+    process noise covariance, we can do order 15 on the car tracking model at least."""
+    regression_problem, statespace_components = filtsmooth_zoo.car_tracking(
+        model_ordint=14,
+        timespan=(0.0, 1e-10),
+        step=1e-12,
+        forward_implementation="sqrt",
+        backward_implementation="sqrt",
+        _process_noise_damping=1e-14,
```

Comment:
I am surprised that a nugget of `1e-14` does anything at all.

Do you think it necessary to establish an entirely new kwarg, solely for the case that somebody wants to use a 14 times integrated model? 😄

Reply:
Well, I dont want to stretch the word "necessary" too much here... :) I just thought it was cool to get these high orders! ;) 
And it is a "hidden" kwarg that defaults to 0, so I thought it doesnt hurt too much. Though if you find it annoying I am open for talking about it :)

Reply:
Well maybe I am biased, but I quickly get confused when a method offers me to tweak every part of the method, and especially if the documentation takes half an hour to read (not that this would be the case here!) So I often prefer clarity over control, given that often, the kwargs are only used in very special cases. Do hidden kwargs appear in the docs?